openbb_sdk/extensions/charting/pyproject.toml

```diff
@@ -12,12 +12,12 @@ openbb-core = { path = "../../sdk/core" }
 scipy = "^1.10.0"
 plotly = "^5.15.0"
 statsmodels = "^0.14.0"
-reportlab = "^4.0.4"
+reportlab = "<4" #Pinning due to build error on MacOs
 pywry = "^0.5.13"
 svglib = "^1.5.1"
 aiohttp = "^3.8.4"
 nbformat = "^5.9.0"
-pandas = ">1.5.3"
```

Comment:
the caret ^ here would pin pandas to 1.X.X, is it what you actually want

Reply:
Isn't this going to get extremely complex and messy when each extension has its own dependency list that will override the already installed versions defined in `openbb_core`?

Reply:
exactly @deeleeramone . cleaning this up is what we need to do